addons/sys_rack/fnc_rackListChildrenActions.sqf

```diff
@@ -20,10 +20,10 @@ params ["_target"];
 private _actions = [];
 
 private _racks = [_target, acre_player] call FUNC(getAccessibleVehicleRacks);
-
 {
     _racks pushBackUnique _x;
 } forEach ([_target, acre_player] call FUNC(getHearableVehicleRacks));
+private _radios = (_racks apply {[_x] call FUNC(getMountedRadio)}) select {_x != "" && {[_x, acre_player] call FUNC(isRadioAccessible)}};
```

Comment:
Using `isRadioAccessible` will render the check {_x in ACRE_HEARABLE_RACK_RADIOS} obsolete since lines 50, 52, 60 and 62 since it will always be []